bin/rbdigital_availability_monitor

```diff
@@ -14,12 +14,5 @@ sys.path.append(os.path.abspath(package_dir))
 
 from core.scripts import RunCollectionMonitorScript
 from api.oneclick import OneClickCirculationMonitor
-from api.oneclick import MockOneClickAPI
-
-from api.config import Configuration
-Configuration.load()
-
-library_id = Configuration.integration(Configuration.ONECLICK_INTEGRATION).get("library_id", None)
-print "Updating availability data of library #%s" % library_id
```

Comment:
This code was left over from when there could only be one RBdigital collection on a server, and it no longer worked.